### ISSUE TITLE
Outdated documentation

### ISSUE THREAD

**Issue by axil** [dhruvasagar/vim-table-mode#63]
Hi, Dhruva

First of all, thanks for your wonderful plugin!

But the documentation is somewhat outdated.

And I spent quite a huge amount of time figuring why the text instructions given on the official page (http://www.vim.org/scripts/script.php?script_id=4501):

> As soon as you type it on a new line (ignores indentation) the script gets to work on creating a table around it.

don't work for me. They imply that *something* should happen after I type it on a new line. I did the `:TableModeEnable` thing, then entered the <Bar> symbol - and nothing happened!

The video on the github page https://github.com/dhruvasagar/vim-table-mode shows *what* should happen: that the very first `<Bar>` gets immediately surrounded by two pluses in the preceding and the following lines. This *didn't happen* for me.

I spent about an hour trying to get the behaviour from your code (now I understand quite well all the nuts and bolts of the plugin :) ) until I realized that it just does some other way now. 

By trial-and-error I managed to understand how.

Then I accidentally found your other video on youtube http://www.youtube.com/watch?v=9lVQ0VJY3ps - and that explained to me that it's not me who's doing it wrong, but that's the plugin behaviour that has changed. 

---------------
TL;DR
Would you please update the phrase I quoted in the documentation and substitute the video to the more recent link I've given.

**Comment by dhruvasagar** [comment]
Hi,

Thanks for pointing it out, please accept my apologies for the wasted time.

The documentation in the README is fairly accurate, I will give that another look, specifically at what you pointed out. Thanks for that.

The videos (including the second one) are actually a bit outdated and the plugin has had changes in it's design based on community feedback in order to be less intrusive with people's regular vim setup. And also for better support for creating flexible tables as per different conventions like Pandoc & Github tables.

Hence I will try and produce a newer video too once I get the time.

However, I would encourage you to open issues whenever you feel something is hard to figure out or get working because that is always high in my priority, to make the plugin as user friendly as possible.

**Comment by axil** [comment]
The second one is much more accurate at least in respect to what to expect and how to start using the plugin. 

IMHO it's much better to put it into documentation right now than to postpone until a better video is ready.

Anyway the amount of time I spent on figuring out how it works is much less than if I had decided to write it myself :)

**Comment by dhruvasagar** [comment]
yea I thought I had both of the videos linked in the README, i'll update that

**Comment by dhruvasagar** [comment]
I have updated it, but it's old :)

The borders aren't created on the fly anymore, you can create them on an as need basis, you just need to hit `||` on an empty line to create a border line. There is also decent support for headers, which is basically the first line separated by a border line.

The docs themselves are fairly accurate, I would highly recommend reading through them rather than the video. I'll try to updated that soon.

**Comment by axil** [comment]
For some reason vim didn't see the 'docs' directory so `:h table-mode` didn't work for me.

Now I've opened it and it is written very well.

Ironically, by that time I've already written an introduction as I see it. 

So I've reread what I've written and found that it can serve a good job as an introduction for the newcomers.

**Comment by axil** [comment]
See Updated readme #65 pull request
